code/src/containers/forms/product-form/components/additional-info/AdditionalInfo.tsx

```diff
@@ -0,0 +1,110 @@
+import { Controller, ControllerRenderProps } from "react-hook-form";
+
+import { productCategories } from "@/containers/forms/product-form/ProductForm.constants";
+import {
+  ProductFormAdditionalInfoSectionProps,
+  ProductFormValues
+} from "@/containers/forms/product-form/ProductForm.types";
+
+import AppBox from "@/components/app-box/AppBox";
+import AppCheckbox from "@/components/app-checkbox/AppCheckbox";
+import AppFormHelperText from "@/components/app-form-helper-text/AppFormHelperText";
+import AppInput from "@/components/app-input/AppInput";
+import AppMenuItem from "@/components/app-menu-item/AppMenuItem";
+import AppSelect from "@/components/app-select/AppSelect";
+import AppTypography from "@/components/app-typography/AppTypography";
+
+const AdditionalInfo = ({
+  register,
+  errors,
+  control
+}: ProductFormAdditionalInfoSectionProps) => {
+  const categoryErrorElement = errors.category && (
+    <AppFormHelperText className="product-form__category-select-helper" error>
+      {errors.category.message}
+    </AppFormHelperText>
+  );
+
+  const selectControllerRenderFunction = ({
+    field: handlers
+  }: {
+    field: ControllerRenderProps<ProductFormValues>;
+  }) => (
+    <>
+      <AppSelect
+        fullWidth
+        label="productForm.inputLabel.category"
+        inputProps={{
+          className: "product-form__category-select"
+        }}
+        error={Boolean(errors.category)}
+        data-testid="product-form-category-select"
+        {...handlers}
+      >
+        {productCategories.map((item) => (
+          <AppMenuItem value={item.id} key={item.id}>
+            <AppTypography
+              className="product-form__category-select-label"
+              translationKey={item.label}
+            />
+          </AppMenuItem>
+        ))}
```

Comment:
Move above return statement for the same reason

Reply:
Thank you, fixed